src/System.Memory/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.1.1</AssemblyVersion>
+    <AssemblyVersion>4.0.1.2</AssemblyVersion>
     <PackageVersion>4.5.1</PackageVersion>
```

Comment:
nit: not from your commit, but we should either use this value or delete it.

Reply:
Do you feel it should be included as part of this PR or that it should be left "as-is" in the servicing branch here and removed in master?